client/asset/eth/eth.go

```diff
@@ -295,7 +295,7 @@ type ethFetcher interface {
 	sendSignedTransaction(ctx context.Context, tx *types.Transaction) error
 	sendTransaction(ctx context.Context, txOpts *bind.TransactOpts, to common.Address, data []byte) (*types.Transaction, error)
 	signData(data []byte) (sig, pubKey []byte, err error)
-	syncProgress(context.Context) (*ethereum.SyncProgress, error)
+	syncProgress(context.Context) (progress *ethereum.SyncProgress, bestHeaderUNIXTime uint64, err error)
```

Comment:
The providers fetch the block header anyway, so this is an optimization that cuts off one header fetch for providers.